src/seedu/addressbook/AddressBook.java

```diff
@@ -1092,7 +1138,8 @@ private static String getUsageInfoForAllCommands() {
                 + getUsageInfoForDeleteCommand() + LS
                 + getUsageInfoForClearCommand() + LS
                 + getUsageInfoForExitCommand() + LS
-                + getUsageInfoForHelpCommand();
+                + getUsageInfoForHelpCommand() + LS
```

Comment:
you can rename `LS` to something more meaningful. please pay attention to such locations for improving the code.